src/software/world/robot_capabilities.cpp

```diff
@@ -0,0 +1,36 @@
+#include "software/world/robot_capabilities.h"
```

Comment:
why was this file added?

Reply:
Whoops, was carry-over from another architecture I tried where the goalie became a robot capability. Removed.

Reply:
Ok, actually removed now.